utils/kubernetes/get-manifests.go

```diff
@@ -0,0 +1,50 @@
+package kubernetes
+
+import (
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/cli/values"
+	"helm.sh/helm/v3/pkg/getter"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+func GetManifestsFromHelm(client *Client, url string) (string, error) {
```

Comment:
@Revolyssup A few comments explaining what is happening here would help.
* How is this function getting manifests from helm charts?
* Is this also applying helm charts? If so, why is there a need to do so?

Reply:
If this is applying helm charts for some reason, can this reuse https://github.com/layer5io/meshkit/blob/cbaca9a35a40eb5bc9930bbbabc6bbc2498b2fd3/utils/kubernetes/apply-helm-chart.go#L220?

Reply:
> @Revolyssup A few comments (comments in code) explaining what is happening here would help.
> 
> * How is this function getting manifests from helm charts?
> * Is this also applying helm charts? If so, why is there a need to do so?

Ans 1- I went to the source code of helm and tried to find where exactly do they populate the config with manifests. The manifests are inside the release struct which is returned by the Run function on *action.Install. So I take out the release from their and manifests are inside the release.
Ans 2- No it doesn't, I have set the "dry run" as true before executing the Run function. Although it needs to be able to communicate to kube api server(I dont know for what reason), it doesn't sends the manifests to the k8s. So no , it doesn't actually apply those helm charts.

Reply:
> If this is applying helm charts for some reason, can this reuse
> 
> https://github.com/layer5io/meshkit/blob/cbaca9a35a40eb5bc9930bbbabc6bbc2498b2fd3/utils/kubernetes/apply-helm-chart.go#L220
> 
> ?
Yeah I also initially thought to use this function as utkarsh has already included a dry run flag in the config but didn't use because-
This function doesn't return any release struct or manifests which we actually need. Getting logs out of here was a bit tricky for me.It seemed easier to write 30-40 lines of code. Although writing this function, I used ApplyHelmChart as reference as well as helm source code.